data-azure-cosmos/src/main/java/io/micronaut/data/cosmos/annotation/ETag.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.data.cosmos.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Field annotation indicating that entity field should be used as a document version or _etag in Azure Cosmos Db.
+ *
+ * @author radovanradic
+ * @since 3.8.2
```

Comment:
I have set since to 3.8.2 since I suppose this is the next version.

Reply:
The next minor should be 3.9.0

Reply:
Makes sense, updated.